payment-service/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>3.1.2</version>
+        <version>3.1.1</version>
```

Comment:
give arguments for 3.1.1 version or change back to 3.1.2

Reply:
done